store/src/java/com/zimbra/cs/filter/jsieve/EditHeaderExtension.java

```diff
@@ -516,8 +516,8 @@ public void commonValidation() throws SyntaxException {
             ZimbraLog.filter.info("No comparator type provided, so setting to default %s", ComparatorNames.ASCII_CASEMAP_COMPARATOR);
         }
         // relational comparator must be available with numeric comparison
-        if (this.comparator.equals(I_ASCII_NUMERIC) && !(this.countTag || this.valueTag)) {
-            throw new SyntaxException(":value or :count not found for numeric operation.");
+        if (this.comparator.equals(I_ASCII_NUMERIC) && !(this.countTag || this.valueTag || this.is)) {
+            throw new SyntaxException(":value or :count or :is not found for numeric operation.");
```

Comment:
should change message to "no valid comparator (:value, :count or :is) found for numeric operation."